support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -750,6 +777,66 @@ export function Checkout() {
 											/>
 										</>
 									)}
+
+									{paymentMethod === 'DirectDebit' && (
+										<DirectDebitForm
+											countryGroupId={countryGroupId}
+											accountHolderName={accountHolderName}
+											accountNumber={accountNumber}
+											accountHolderConfirmation={accountHolderConfirmation}
+											sortCode={sortCode}
+											recaptchaCompleted={false}
+											updateAccountHolderName={(name: string) => {
+												setAccountHolderName(name);
+											}}
+											updateAccountNumber={(number: string) => {
+												setAccountNumber(number);
+											}}
+											updateSortCode={(sortCode: string) => {
+												setSortCode(sortCode);
+											}}
+											updateAccountHolderConfirmation={(
+												confirmation: boolean,
+											) => {
+												setAccountHolderConfirmation(confirmation);
+											}}
+											recaptcha={
+												<Recaptcha
+													// We could change the parents type to Promise and uses await here, but that has
+													// a lot of refactoring with not too much gain
+													onRecaptchaCompleted={(token) => {
+														setRecaptchaToken(token);
+														void fetch(
+															'/stripe/create-setup-intent/recaptcha',
+															{
+																method: 'POST',
+																headers: {
+																	'Content-Type': 'application/json',
+																},
+																body: JSON.stringify({
+																	isTestUser,
+																	stripePublicKey,
+																	token,
+																}),
+															},
+														)
+															.then((resp) => resp.json())
+															.then((json) =>
+																setStripeClientSecret(
+																	(json as Record<string, string>)
+																		.client_secret,
+																),
+															);
+													}}
+													onRecaptchaExpired={() => {
+														// no-op
```

Comment:
Eventually we'll want to call `setRecaptchaToken` here and (un)set the `recaptchaToken` when it expires.